### ISSUE TITLE
Have there been changes to the SDXL LoRA trainer?

### ISSUE THREAD

**Issue by eliohead** [TheLastBen/fast-stable-diffusion#2987]
Hello,
Last summer, I created Loras for SDXL using the Fast Stable Diffusion notebook with unet learning rates in the range of 1e-4. I consistently achieved good results.

Recently, I tried creating LoRAs again using the same notebook, but I noticed that using a learning rate of 1e-4 now always results in a NaN error. This also happens for any learning rate above 5e-5. It seems like the learning rate behavior is much more sensitive now, like a full finetuning instead of a Lora.

I'm not sure if there have been any changes to the notebook or the training process since last summer that might explain this.
